index.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio Setup</title>
+    <link rel="stylesheet" href="style.css">
+    <!-- font awesome icons -->
+    <script src="https://kit.fontawesome.com/25935046d9.js" crossorigin="anonymous"></script>  
+</head>
+<body>
+    <div class="container">
```

Comment:
- [ ] Please let's get rid of this `div` wrapper so that we can completely differentiate the toolbar section from the headline section. What do you think?

Reply:
Hi @FrederickMih,

Trust you are well. I will like to appreciate your time given to review my project. However, I will like to seek more clarification probably outside the review process on the comment you made on my project

1. About the div tag in the above header, I was not cleared enough as to the view you shared on removing it because that div tag was used to contain very important properties regarding the background set for the project